web/src/test-utils.js

```diff
@@ -30,8 +31,9 @@ const InstallerProvider = ({ children }) => {
   return <InstallerClientProvider client={client}>{children}</InstallerClientProvider>;
 };
 
-const installerRender = (ui, options = {}) => {
-  return render(ui, { wrapper: InstallerProvider, ...options });
-};
+const installerRender = (ui, options = {}) => ({
+  user: userEvent.setup(),
+  ...render(ui, { wrapper: InstallerProvider, ...options })
+});
```

Comment:
I do not have a strong preference here right now. We can also provide a separated `user` util and use it where needed instead of including it here. What do you think?

```js
const setupUser = () => userEvent.setup();

const installerRender = (ui, options = {}) => {
  return render(ui, { wrapper: InstallerProvider, ...options });
};

export { 
  setupUser,
  installerRender
}
```

And then, 

```js
import { setupUser, installerRender } from 'test-utils';

it("looks better for you", async () => {
  installerRender(<DInstaller />);  
  
  const user = setupUser();
 
  await user.click("Install");
});
```